src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ *
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ *
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  function editStyles(prev, style) {
+    const separatorIndex = style.indexOf(':');
+
+    return {
+      ...prev,
+      [style.slice(0, separatorIndex).trim()]: style.slice(separatorIndex + 1)
+        .trim(),
+    };
+  }
```

Comment:
Move this function to the outside of convertToObject function, to avoid of creating it on each call.